AutoRest/Generators/Ruby/Ruby/TemplateModels/ModelTemplateModel.cs

```diff
@@ -124,6 +124,22 @@ public ModelTemplateModel(CompositeType source, ISet<CompositeType> allTypes)
             PropertyTemplateModels = new List<PropertyTemplateModel>();
             source.Properties.ForEach(p => PropertyTemplateModels.Add(new PropertyTemplateModel(p)));
 
+            if (!string.IsNullOrEmpty(source.PolymorphicDiscriminator))
```

Comment:
@vishrutshah This is the section where we bring the properties from the parent into each model, which we used not to do? (what we talked yesterday?)